atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraReloadableKvsConfig.java

```diff
@@ -237,7 +244,14 @@ public String type() {
 
     @Override
     public int concurrentGetRangesThreadPoolSize() {
-        return config.concurrentGetRangesThreadPoolSize();
+        return chooseConfig(
```

Comment:
This still isn't quite right. We may use install `servers` but runtime `concurrentGetRangesThreadPoolSize`/`pool`. 

For example, if an existing user took this upgrade they would have a `concurrentGetRangesThreadPoolSize` value of 0.